hathor/merged_mining/bitcoin_rpc.py

```diff
@@ -23,6 +23,12 @@ async def get_block_template(self, *, rules: List[str] = ['segwit'],
         """
         raise NotImplementedError
 
+    @abstractmethod
+    async def verify_block_proposal(self, *, block: bytes) -> Optional[str]:
+        """ Method for the [GetBlockTemplate call](https://bitcoin.org/en/developer-reference#getblocktemplate).
```

Comment:
Couldn't find this link, just https://developer.bitcoin.org/reference/rpc/getblocktemplate.html

Reply:
I copied the link from `get_block_template`, it was working before, but I think I'll have to fix the other links too.

Reply:
Fixed.